inspirehep/modules/records/serializers/schemas/json/literature/common/reference_item.py

```diff
@@ -49,21 +55,33 @@ class ReferenceItemSchemaV1(Schema):
 
     @pre_dump(pass_many=True)
     def filter_references(self, data, many):
-        reference_records = self.get_resolved_references_by_control_number(data)
+        reference_records = self.get_resolved_references_by_control_number(
+            data)
 
         if not many:
             return self.get_resolved_reference(data, reference_records)
 
         references = []
         for reference in data:
-            resolved_reference = self.get_resolved_reference(reference, reference_records)
+            resolved_reference = self.get_resolved_reference(
+                reference, reference_records)
             references.append(resolved_reference)
         return references
 
+    @pre_dump
+    def force_each_collaboration_to_be_object(self, data):
```

Comment:
it's not a best practice to what to ``pre_dump`` better to add the logic to two different methods https://marshmallow.readthedocs.io/en/3.0/extending.html?highlight=pre_dump#pre-post-processor-invocation-order (Warning part)

Reply:
warning is for if you care about the order which we don't.